app/mailers/authentication_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class AuthenticationMailer < Mail::Notify::Mailer
+  GENERIC_NOTIFY_TEMPLATE = '2744ea53-34f1-431f-8173-8388fadd826a'.freeze
```

Comment:
This shouldn't need to be repeated here. Could it be pushed up into something it shares in common with `tt_application_mailer`?

Reply:
Makes sense! We've made `ApplicationMailer` inherit from `Mail::Notify::Mailer` as all emails will need to use Notify. Then we made all our mailers inherit from `ApplicationMailer` and store `GENERIC_NOTIFY_TEMPLATE`.